src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_padding.cpp

```diff
@@ -249,15 +250,16 @@ void prepare_padding::run(program& p) {
 
         auto needed_padding = get_needed_padding_for_convolution(node);
 
-        auto& input = node.get_dependency(0);
-        // WA to add reorder between MVN and Conv because Conv need input data with padding but MVN opt kernel with default format does not support padding.
+        // WA to add reorder between MVN and Convolution or between col2im and Convolution
+        // because Conv need input_node data with padding but MVN opt with default format and col2im do not support padding.
```

Comment:
This code looks good as-is, but did you have a chance to review padding support from col2im? is it complicated?